_posts/2016-05-09-twis-62.md

```diff
@@ -0,0 +1,51 @@
+---
+layout:     post
+title:      This Week In Servo 62
+date:       2016-05-09 00:30:00
+summary:    What's up with Servo for the week of 02 May 2016 - 09 May 2016
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2016-05-02..2016-05-09+user%3Aservo), we landed 133 PRs in the Servo organization's repositories.
+
+### Planning and Status
+
+Our overall [roadmap](https://github.com/servo/servo/wiki/Roadmap) and [quarterly goals](https://docs.google.com/document/d/1JMOtVkRtb-s7auoQdnX810HGglkMK054LTXOo0_rdrU/pub) are available online.
+
+This week's status updates are [here](http://statusupdates.dev.mozaws.net/project/servo).
+
+### Notable Additions
+ 
+ - ms2ger [added](https://github.com/servo/rust-selectors/pull/86) support for using Gecko's string atoms in rust-selectors, which is a major performance issue for reusing Servo's style system in Firefox
+ - autrilla [changed](https://github.com/servo/servo/pull/11067) our build driver script, `mach`, to display more information when there is a virtualenv or python failure
+ - larsberg [increased](https://github.com/servo/saltfs/pull/366) the number of parallel processes in use on our Mac builders which, in conjunction with some other system tuning, reduces the OSX builder times to around 30 minutes
```

Comment:
Only the test-css portion, not the test-wpt portion. test-wpt is where we saw significant speedups; test-css only got us a couple of minutes.